<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneInfoWidget</class>
 <widget class="QWidget" name="GeneInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QLabel" name="labelLen">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Length</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelTypeOut">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelChr">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Chr</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelStrandOut">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelNameOut">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelStrand">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Strand</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelLenOut">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPos">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelPosOut">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelChrOut">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>(NULL)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAnno">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-top-width:4px;
    border-color:#353b48;
    background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Annotation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textAnno">
     <property name="styleSheet">
      <string notr="true">QTextBrowesr{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
